content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,119 @@
+    $ kubectl cluster-info --context kind-istio-testing
+    Kubernetes master is running at https://127.0.0.1:32773
+    KubeDNS is running at https://127.0.0.1:32773/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+    {{< /text >}}
+
+1.  In case you already have another local cluster, you must set kind as current cluster in order to use it.
+    View local cluster configuration using following command
+
+    {{< text bash >}}
+    $ kubectl config view
+    {{< /text >}}
+
+    Make sure that the `current-context` is set to `kind-istio-testing`. If it is set to another cluster like minikube,
+    You can set kind as a current cluster using the following command.
+
+    {{< text bash >}}
+    $ kubectl config use-context kind-istio-testing
+    kubectl config use-context kind-istio-testing
+    {{< /text >}}
+
+1.  When you are done experimenting and you want to delete existing cluster, use the following command:
+
+    {{< text bash >}}
+    $ kind delete cluster --name istio-testing
+    {{< /text >}}
+
+## Setup Dashboard UI for kind
+
+kind does not have in built dashboard UI like minikube. But you can still setup and view your cluster.
+Follow these instruction to setup dashboard UI for kind.
+
+1.  The dashboard UI is not deployed by default. To deploy it, run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml
+    {{< /text >}}
+
+1.  Verify that the dashboard is deployed and running.
+
+    {{< text bash >}}
+    $ kubectl get pod -n kubernetes-dashboard
+    NAME                                         READY   STATUS    RESTARTS   AGE
+    dashboard-metrics-scraper-76585494d8-zdb66   1/1     Running   0          39s
+    kubernetes-dashboard-b7ffbc8cb-zl8zg         1/1     Running   0          39s
+    {{< /text >}}
+
+1.  Create `ClusterRoleBinding` to provide admin access to the newly created cluster.
+
+    {{< text bash >}}
+    $ kubectl create clusterrolebinding default-admin --clusterrole cluster-admin --serviceaccount=default:default
+    {{< /text >}}
+
+1.  In order to login to the dashboard UI you need a token. Use following command to store token in variable.
+
+    {{< text bash >}}
+    $ token=$(kubectl get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='default')].data.token}"|base64 -d)
+    {{< /text >}}
+
+    Display token using the echo command and copy it so you can use it to login to dashboard.
+
+    {{< text bash >}}
+    $ echo $token
+    {{< /text >}}
+
+1.  You can access dashboard using the kubectl command-line tool by running the following command:
+
+    {{< text bash >}}
+    $ kubectl proxy
+    Starting to serve on 127.0.0.1:8001
+    {{< /text >}}
+
+    Kubectl will make dashboard available at [localhost:8001](http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/).
```

Comment:
I thought I would be nice and just enter `localhost:8001` in the browser and that didn't get me to the dashboard. Clicking this link did, but that's because of the additions to the URL. 

Not sure of the best way to fix this, but maybe change the localhost:8001 as the link text to `Kubernetes dashboard`. Or maybe just have a more explicit link.

Reply:
You are right, `localhost:8001` is misleading. I tired putting whole URL but linter fails. I can update it to simple `Kubernetes dashboard` and ask user to click on it.